apps/cfp/tasks.py

```diff
@@ -150,3 +150,31 @@ def create_tags(tags_to_create):
 
     db.session.commit()
     app.logger.info(f"Successfully created {tags_created} new tags.")
+
+
+@cfp.cli.command(
+    "delete_tags",
+    help=f"Delete tags from the Database. Tagged proposals will have the tags removed.",
+)
+@click.argument("tags_to_delete", required=True, nargs=-1)
+def delete_tags(tags_to_delete):
+    """Delete tag list"""
+    tags_deleted = 0
+    for tag_name in tags_to_delete:
+        tag = Tag.query.filter_by(tag=tag_name).one_or_none()
+        if not tag:
+            app.logger.info(f"Couldn't find tag: '{tag_name}' exiting.")
+            return
+
+        if tag.proposals:
+            tagged_proposals = [p.id for p in tag.proposals]
+            app.logger.info(
+                f"'{tag_name}' will be removed from the proposals with ids: {tagged_proposals}"
```

Comment:
Nitpicky: would it be helpful to mention each proposal `title` as well?  (thinking: could provide a useful sense-check during tag removal)

Reply:
probably not? This will hopefully be a pretty rare manual task (after I do what I need to do which is update the current list of tags to match those selected by the content team) and untagged proposals can be found via the UI. 

In theory deleting a tag will only happen when it's not useful: i.e. has very few (probably no) members OR so many members it's pointless.

Reply:
Okie doke, just checking.  Seems reasonable!